varinfo/cmr_search.py

```diff
@@ -134,3 +143,41 @@ def download_granule(granule_link: str,
         # Custom exception for error from `requests.get`
         raise GranuleDownloadException(
             str(requests_exception)) from requests_exception
+
+
+def get_edl_token_from_launchpad(launchpad_token: str,
```

Comment:
@owenlittlejohns for consistency do you want me to change this too `auth_header` too

Reply:
I think it makes sense for this to still be `launchpad_token`, because this function is only called by `get_edl_token_header` if you have determined that the auth header is a LaunchPad token (and so this function has an unambiguous input).